scripts/jest/test-framework-setup.js

```diff
@@ -7,33 +7,25 @@ if (process.env.REACT_CLASS_EQUIVALENCE_TEST) {
 } else {
   var env = jasmine.getEnv();
 
-  var callCount = 0;
-  var oldError = console.error;
-  var newError = function() {
-    callCount++;
-    oldError.apply(this, arguments);
-  };
-
-  console.error = newError;
-
   // TODO: Stop using spyOn in all the test since that seem deprecated.
   // This is a legacy upgrade path strategy from:
   // https://github.com/facebook/jest/blob/v20.0.4/packages/jest-matchers/src/spyMatchers.js#L160
   const isSpy = spy => spy.calls && typeof spy.calls.count === 'function';
 
   env.beforeEach(() => {
-    callCount = 0;
     jasmine.addMatchers({
       toBeReset() {
         return {
           compare(actual) {
             // TODO: Catch test cases that call spyOn() but don't inspect the mock
             // properly.
-            if (actual !== newError && !isSpy(actual)) {
+            if (actual.__callCount === undefined && !isSpy(actual)) {
```

Comment:
I figured it out. It was asserting we didn't overwrite `console.error` explicitly in a test. Changed the logic to verify the same thing (but for `warn` too).